### PR TITLE
refactor: cleanup

### PR DESCRIPTION
**Pull request by SirWindfield**

### CODE REVIEW

**Comment by k0pernicus** [comment]
looks good to me too, thanks @SirWindfield :)

**Comment by k0pernicus** [comment]
Looks good to me too, thanks @SirWindfield 🎉   
However, it could be great to "merge" your changes from #232 please 👍

**Comment by SirWindfield** [comment]
Ye I can combine the PRs if that is what you want. Not sure what the conflict is right now, lemme take a look.

File: src/lib.rs [hunk 1]
```diff
@@ -44,25 +44,23 @@ use structopt::StructOpt;
...
+    #[structopt(name = "generate", visible_alias = "gen")]
     Generate(Args),
-    #[structopt(name = "gen")]
-    Gen(Args),
 }
 
 #[derive(Debug, StructOpt)]
 pub struct Args {
-    #[structopt(long = "git")]
+    #[structopt(long)]
```
[the remark this answers is not in the document]
Comment (k0pernicus) [reply src/lib.rs]: Thanks!

File: src/lib.rs [hunk 2]
```diff
@@ -44,25 +44,23 @@ use structopt::StructOpt;
...
     git: String,
     // Branch to use when installing from git
-    #[structopt(long = "branch")]
+    #[structopt(long)]
```
[the remark this answers is not in the document]
Comment (k0pernicus) [reply src/lib.rs]: Thanks!

**Comment by k0pernicus** [comment]
Thanks again @SirWindfield 🚀

**Pull request merged**